script.js

```diff
@@ -1,12 +1,273 @@
+  theLink.setAttribute('rel', 'noreferrer noopener');
+  
+  tvmInfoDiv.appendChild(theLink);
+  tvMazeInfo.appendChild(tvmInfoDiv);
 }
 
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
   rootElem.textContent = `Got ${episodeList.length} episode(s)`;
 }
 
-window.onload = setup;
+function showAll(setup_options) {
+       for (let i = 0; i < allEpisodes.length; i++)
+       {
+              createAllEpisodes(i);
+              // at the same time set up the dropdown option menu
+              if (setup_options) { 
+                          let option = document.createElement('option');
+                          option.value = fetchEpisodeSeason_Suffix(allEpisodes[i]); // EG 'S01E01'
+                          option.text = fetchEpisodeSeason_Full(allEpisodes[i]); // EG Winter is Coming - S01E01
+                          selectMenu.appendChild(option);
+                                }           
+       }
+       displayMessage[0].innerText = `Displaying ${allEpisodesTotal}/${allEpisodesTotal} episodes.`;
+}
+
+function createAllEpisodes(index) {
+    const episodeDiv = document.createElement('div'); // main 'div' to append to
+    const source = allEpisodes[index]; // Entire Episode Entry
+    episodeDiv.setAttribute("id",fetchEpisodeSeason_Suffix(source)); // EG S01E01
+
+    let episodeInfo = fetchEpisodeSeason_Full(source); // EG Winter is Coming - S01E01
+    const text = document.createElement('h1');
+    text.classList.add('episode-title');
+    text.innerText = episodeInfo;
+    episodeDiv.appendChild(text); // add the title
+
+    const imageDiv = document.createElement('div'); // div for an image
+    imageDiv.classList.add('image');
+    episodeDiv.appendChild(imageDiv);
+
+    const img = document.createElement('img');
+    img.src = source.image.medium;
+    img.alt = episodeInfo;
+    episodeDiv.appendChild(img); // add the image
+
+    // Remove all <p></p> then add the text using <span>
+    const summary = handle_paragraph(source.summary);
+    episodeDiv.appendChild(summary); // add the summary
+
+    // add to main content
+    mainDisplayDiv.appendChild(episodeDiv); 
+
+    if ((index+1) % 3 === 0) // MAKE THE 4TH COLUMN A GAP BY CREATING AN EMPTY DIV
+    {
+           const blankColumn = document.createElement('div'); // main 'div' to append to
+           mainDisplayDiv.appendChild(blankColumn);
+    }
+
+}
+
+// EG Winter is Coming - S01E01
+function fetchEpisodeSeason_Full(info) {
+        return `${info.name} - S${String(info.season).padStart(2,"0")}E${String(info.number).padStart(2,"0")}`;
+}
+
+// EG 'S01E01'
+function fetchEpisodeSeason_Suffix(info) {
+        return `S${String(info.season).padStart(2,"0")}E${String(info.number).padStart(2,"0")}`;
+}
+
+
+// Instead of using innerHtml, remove the <p></p> and use innerText instead
+function handle_paragraph(text) {
+      text = text.replace(/<\/?p>/ig,""); // remove any <p> </p>
```

Comment:
There is also a couple of paragraphs, where you need to remove <br>. Check Hardhome - S05E08 and The Gift - S05E07